src/types/index.ts

```diff
@@ -0,0 +1,38 @@
+export interface ISource {
+  status: string;
+  sources: ISourceItem[];
+}
+
+export interface ISourceItem {
+  id: string;
+  name: string;
+  description: string;
+  url: string;
+  category: string;
+  language: string;
+  country: string;
+}
+
+export interface INews {
+  articles: IArticles[];
+  status: string;
+  totalResults: number;
+}
+
+export interface IArticles {
+  source: {
+    id: string | null;
+    name: string;
+  };
```

Comment:
I would suggest to move declaration of `source` structure into separate interface as a matter of readability.

Reply:
@Dimash95, pay attention to this comment. It must be fixed before merging PR.